db_model/orm_model.py

```diff
@@ -116,6 +116,7 @@ class BuildAction(Base):
     run_id = Column(Integer,
                     ForeignKey('runs.id', deferrable=True, initially="DEFERRED",
                                ondelete='CASCADE'))
+    # hash of the build command
```

Comment:
Yes I know, I was thinking about it, but that means db schema change which I do not wanted to do right now that is why I put the comment there.
With v6.0 there will be more schema changes  #343 and we definitely should change it.
Is it enough if we change it for 6.0?